pyproject.toml

```diff
@@ -44,13 +44,6 @@ pymongo = {version = "*", optional = true}
 # extra [redis]
 redis = {version = "*", optional = true}
 
-# extra [docs]
```

Comment:
Was seeing sphinx installed in dependent packages. Maybe because `sphinx_autodoc_typehints`, is not included in docs extra below? but in any case, it makes less sense to me to maintain 2 sources of truth when we **need** the requirements.txt inside the docs/ folder, and we don't really need this.